docs/autogenerated-menu/Our Identity/02-Consultancy/02-Default Delivery Process/01-Overview/01-overview.md

```diff
@@ -3,27 +3,38 @@ title: Default Delivery Process Overview
 ---
 # Default Delivery Process Overview
 
+<!-- Scrap 'Default' - Default implies lack of thought.  I think we can use a better word, or drop it -->
+
 ## Introduction
 At Burendo, we aim to deliver high quality solutions that create real business and user impact. To do this well, we need consistent and effective ways of working that guide our teams on what to do and how best to do it.
+<!-- 'we need' implies we lack it.  reword this.  Should come from a position of confidence -->

```

Comment:
I don't see that implication. I think this is saying we need this and this framework ensures we have it. But open to proposals if you have a preferred rewording.

Reply:
If you read this from a non-Burendo POV - We have > We need.